### PR TITLE
Modify `tasks/CMakeLists.txt` to copy files on build

### PR DESCRIPTION
**Pull request by hartikainen**
See #31. Let me know if there's a better solution for this.

### CODE REVIEW

File: mjpc/tasks/CMakeLists.txt [hunk 1]
```diff
@@ -14,4 +14,10 @@
 
 # Copy the model files to the binary directory to make them available to the
 # built binary.
-file(COPY ${CMAKE_CURRENT_SOURCE_DIR} DESTINATION ${CMAKE_BINARY_DIR})
+add_custom_target(copy_resources ALL
+        COMMAND ${CMAKE_COMMAND} -E copy_directory
+        ${CMAKE_CURRENT_SOURCE_DIR}
+        ${CMAKE_CURRENT_BINARY_DIR}
```
Comment (hartikainen) [inline mjpc/tasks/CMakeLists.txt:20]: Should this be `CMAKE_CURRENT_BINARY_DIR` or `CMAKE_BINARY_DIR`?

**Comment by kevinzakka** [comment]
Can confirm this works for me.

File: mjpc/tasks/CMakeLists.txt (hunk 1 above)
Comment (nimrod-gileadi) [reply mjpc/tasks/CMakeLists.txt:20]: CMAKE_CURRENT_BINARY_DIR makes sense to me.

**Review by nimrod-gileadi — APPROVED** [review]

**Comment by hartikainen** [comment]
Confirmed that this works for me too.

**Pull request merged**